automation/check-patch.packages

```diff
@@ -0,0 +1,4 @@
+git
```

Comment:
Do we need all of these now? Please drop unused dependencies.

Reply:
It was requested by @SchSeba

Reply:
@phoracek I think the only one we can drop is git no?

Reply:
Also I don't see rsync and make used.

Reply:
Ok lets try.
This was copy from the kubevirt repo.